Fitter/TauES/createinputsTES.py

```diff
@@ -9,23 +9,51 @@
 from config.samples_v12 import *
 from TauFW.Plotter.plot.utils import LOG as PLOG
 from TauFW.Fitter.plot.datacard import createinputs, plotinputs
+from TauFW.Fitter.plot.rebinning import rebinning
 import yaml
 
+#nano doc: https://cms-nanoaod-integration.web.cern.ch/autoDoc/NanoAODv12/2022/2023/doc_DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8_Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v2.html
+
+map_wp_to_int = OrderedDict([('againstjet', 
+				OrderedDict([('Loose',  4),
+					     ('Medium', 5),
+					     ('Tight',  6),
+					     ('VTight', 7)])),
+	                    ('againstelectron',
+				OrderedDict([('VVLoose', 2),
+                                             ('Tight', 6),
+                                             ('Loose', 1 )]))
```

Comment:
isn't Loose = 4?

Reply:
Right!

Reply:
ok, fix it please :) (hope it didn't affect SFs you delivered)

Reply:
done